mission_control/include/mission_control/mission.h

```diff
@@ -65,58 +64,31 @@ class Mission
     PAUSED,
     COMPLETE
   };
-  void SetState(MissionState state);
-  MissionState GetState();
-  boost::mutex m_MissionStateLock;
-
-  void ExecuteMission(ros::NodeHandle nh);
-  void AbortMissionWrapper(ros::NodeHandle nh);
 
+  void processMission();
+  void ExecuteMission();
```

Comment:
@patapufate nit: be consistent with style.

Reply:
Fixed!